src/NServiceBus.NHibernate.Tests/SynchronizedStorage/NHibernateOutboxTransactionSynchronizedStorageSessionTests.cs

```diff
@@ -142,5 +134,18 @@ public async Task It_does_not_commit_outbox_transaction_if_callback_throws()
                 Assert.IsNull(savedEntity);
             }
         }
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Code", "PS0018:A task-returning method should have a CancellationToken parameter unless it has a parameter implementing ICancellableContext", Justification = "<Pending>")]
```

Comment:
Something is off that the tests trigger the analyzer. Maybe we need to configure things properly in the editor config file?

Reply:
👍